src/test/java/hudson/security/LDAPEmbeddedTest.java

```diff
@@ -549,6 +548,7 @@ public void validate() throws Exception {
         assertThat("Always report outer kind as OK", validation.kind, is(FormValidation.Kind.OK));
     }
 
+    @Ignore("TODO InitialLdapContext in fact receives typesOnly, yet everything seems to work anyway in Spring (attributes are defined)")
```

Comment:
Spent most of the day trying to figure this out, without success. Not clear whether there is actually something wrong in the plugin, or just the test making unwarranted assumptions.

Reply:
also https://github.com/jenkinsci/acceptance-test-harness/pull/607

Reply:
I'm actually not sure what this test is supposed to test, except judging by the name of the test it checks that environment properties are set correctly. So I don't think it specifically needs to check that you can't login after setting `java.naming.ldap.typesOnly` :) It could test some other value instead.